float64[] time # Vector of times
c2_ros/State[] statemsg # Vector of states
c2_ros/Ctrl[] ctrl # Vector of Ctrls
int32 N # Number of segments in the ctrl trajectory
c2_ros/State finalgoal # Final goal 
string rootname
